.env.development

```diff
@@ -0,0 +1,2 @@
+// suppress "--inspect=" parameter applied automatically by "electron-webpack dev"
+ELECTRON_ARGS=[]
```

Comment:
The `--inspect=` added by `electron-webpack dev` ends up confusing `yargs` when debugging or otherwise running in developer mode.